public/index.html

```
@@ -1335,7 +1354,7 @@ <h1>PTZ Preset Control</h1>
     <div id="preset-grid"><!-- rendered by JS --></div>
   </div>
   <div id="toolbar">
-    <button class="btn-primary" id="btn-scan">Scan All Presets</button>
+    <button class="btn-primary" id="btn-scan">Recapture Preset Range</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)